app/components/elements/DatePicker.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import momentPropTypes from 'react-moment-proptypes';
+import { SingleDatePicker, SingleDatePickerShape } from 'react-dates';
+import NavigateBeforeRoundedIcon from '@material-ui/icons/NavigateBeforeRounded';
+import NavigateNextRoundedIcon from '@material-ui/icons/NavigateNextRounded';
+import CloseRoundedIcon from '@material-ui/icons/CloseRounded';
+import omit from 'lodash/omit';
+import noop from 'lodash/noop';
+import styled from 'styled-components';
+
+import { DatePicker as StyledDatePickerBase } from './InputStyles';
+
+import {
+  colors,
+  fontSizes,
+  radii,
+  shadows,
+  space,
+} from '../../themes/baseTheme';
```

Comment:
Is this more for just dropping the repeated object nest reference (`theme.colors`, `theme.fontSizes` etc) from the default export or for hopeful tree-shaking down the line? I'm fine with it either way, just curious.

Reply:
The first -- just wanted to avoid the nesting.